pkg/kms/localkms/keytemplate_factory_ursa.go

```diff
@@ -0,0 +1,36 @@
+//go:build ursa
+// +build ursa
+
+/*
+Copyright Avast Software. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package localkms
+
+import (
+	tinkpb "github.com/google/tink/go/proto/tink_go_proto"
+
+	clbld "github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/cl/blinder"
+	clsgn "github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/cl/signer"
+	"github.com/hyperledger/aries-framework-go/pkg/kms"
+)
+
+// getKeyTemplate returns tink KeyTemplate associated with the provided keyType.
+func getKeyTemplate(keyType kms.KeyType, opts ...kms.KeyOpts) (*tinkpb.KeyTemplate, error) {
```

Comment:
Shouldn't Ursa build complement normal build instead of replacing it? It seems Ursa build only supports CL keys and nothing else.

Reply:
it'll fallback to `keyTemplate` which contains switch statement for all other keys, if passed `keyType` isn't one of `ursa`s
```
	default:
		return keyTemplate(keyType, opts...)
```